app/components/CapitalProjectsList/CapitalProjectsDrawer.tsx

```diff
@@ -0,0 +1,66 @@
+import { Box, Flex, Heading } from "@nycplanning/streetscape";
+import { Slide } from "@chakra-ui/react";
```

Comment:
How are we using `Slide`?

I don't see us import `@chakra-ui/react` in the package.json

Reply:
Hm, I'm not really sure tbh. I wonder if it's because I never did an install after removing `@chakra-ui/react` from earlier feedback and my local modules remained the same. (I don't think it's coming from streetscape for what it's worth) 

I included `@chakra-ui/transition` now in`package.json`

Reply:
This is likely because I had you remove the dependency in [this comment](https://github.com/NYCPlanning/ae-cp-map/pull/56#discussion_r1709857646) but now you _are_ depending on it again. I'm assuming it still works on your machine because `@chakra-ui/react` is still in your `node_modules`. Adding `@chakra-ui/transition` as a dependency is a good solution for now until we add `Slide` to Streetscape.

Reply:
Nice, just what I had suspected. 

Is there a typical way I could update my local node modules? A reinstall doesn't seem to get rid of ones that are no longer in `package.json`

Reply:
I think to get rid of unwanted modules, you probably would need to delete all the modules ( `rm -rf node_modules` ) and  reinstall them  to get a fresh, clean set of modules.

Reply:
Ooo didn't know about `npm prune` ty

Reply:
I opened to NYCPlanning/ae-streetscape#71 so that we can get the `slide` component from streetscape instead of Chakra. Though, I think we should wait to do said refactor until #71.

@pratishta and @TylerMatteo

Reply:
Thanks for that quick work. And yeah prefer to do refactoring of that nature not-in-this-ticket.